jobserver/models/job_request.py

```diff
@@ -43,17 +43,17 @@ def with_started_at(self):
 class JobRequestManager(models.Manager.from_queryset(JobRequestQuerySet)):
     use_in_migrations = True
 
-    def previous(self, job_request):
-        return (
-            super()
-            .filter(
-                workspace=job_request.workspace,
-                backend=job_request.backend,
-                id__lt=job_request.id,
-            )
-            .order_by("created_at")
-            .last()
+    def previous(self, job_request, succeeded=None):
```

Comment:
If I passed `succeeded=False`, what would this method return? Rather than parameterizing this method, consider adding another method.